Sources/OktaIdx/Internal/Implementations/Version1/Responses/IDXClient+V1ResponseConstructors.swift

```diff
@@ -173,12 +173,12 @@ extension IDXClient.Remediation.Option: IDXHasRelatedObjects {
     internal convenience init?(api: IDXClientAPIImpl, v1 object: V1.Response.Form?) {
         guard let object = object else { return nil }
         self.init(api: api,
-                  rel: object.rel,
+                  rel: object.rel ?? [],
                   name: object.name,
                   method: object.method,
                   href: object.href,
-                  accepts: object.accepts,
-                  form: object.value.map { (value) in
+                  accepts: object.accepts ?? "",
```

Comment:
Instead of passing an empty string, we should make the `IDXClient.Remediation.Option` `accepts` property nullable. Same goes for the `rel` property.

Reply:
Done, sir :)

Reply:
It looks like you're still converting `nil` accepts and `rel` to empty non null-values. I think we should just pass the raw nullable values along, and ensure IDXClient.Remedation.Option has its relevant properties updated to indicate they can be null.

Reply:
Must be done 🙄